Source/WTF/wtf/SystemTracing.h

```diff
@@ -163,10 +163,21 @@ enum TracePointCode {
     WebXRCPFrameStartSubmissionEnd,
     WebXRCPFrameEndSubmissionStart,
     WebXRCPFrameEndSubmissionEnd,
+    DisplayLinkUpdate,
 
     GPUProcessRange = 16000,
     WakeUpAndApplyDisplayListStart,
     WakeUpAndApplyDisplayListEnd,
+
+#if USE(GLIB)
+    GLibPortRange = 20000,
```

Comment:
I think this is a bit confusing, because there's no GLib port and because these are not glib specific events. Maybe we could add a CompositionRange or something like that.

Reply:
They are all gtk/wpe specific, and only used for these ports. That's why I went with USE(GLIB) instead of PLATFORM(GTK) || PLATFORM(WPE).

The only addition here in this patch that is cross-platform is 'DisplayLinkUpdate'.